src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/Utils.cs

```diff
@@ -583,6 +583,18 @@ internal static byte[] ConvertIntToByteArray(int dwInput)
             return rgbOutput;
         }
 
+        internal static int ConvertByteArrayToInt(byte[] input)
```

Comment:
Yeah, I considered `BigInt`, but that's always little-endian. The spec says it should be big-endian, from memory, and I can never remember which sets of bytes should be swapped to convert them (Int32 is 4 bytes, but `ConvertIntToByteArray` seems to make a point of not including the leading empty bytes).

I saw an existing implementation and figured it was less risky than rolling my own.